docs/data/sdks/go/ampli.md

```diff
@@ -106,9 +107,8 @@ ampli.Instance.Load(ampli.LoadOptions{
 
 | <div class ="big-column">Arg of LoadOptions</div> | Description |
 |-|-|
-|`Environment`| Required. String. Specifies the environment the Ampli Wrapper is running in. For example,  `EnvironmentProduction` or `EnvironmentDevelopment`. Create, rename, and manage environments in Amplitude Data.<br /><br />Environment determines which API token is used when sending events.<br /><br />If a `Client.ApiKey` or `Client.Instance` is provided, `Environment` is ignored, and can be omitted.|
+|`Environment`| Required. String. Specifies the environment the Ampli Wrapper is running in. For example,  `EnvironmentProd` or `EnvironmentDev`. Create, rename, and manage environments in Amplitude Data.<br /><br />Environment determines which API token is used when sending events.<br /><br />If a `Client.ApiKey` or `Client.Instance` is provided, `Environment` is ignored, and can be omitted.|
```

Comment:
I need to double check on the default names, but I think this should be: `EnvironmentProduction` or `EnvironmentDevelopment`

Reply:
Fixed

Reply:
Thanks @falconandy. I confirmed with the Data team 

> it currently depends if it’s a migrated project or a not - production for new projects